conf/gulp-tasks/bundle/bundletaskfactory.js

```diff
@@ -41,11 +43,15 @@ class BundleTaskFactory {
       case BundleType.MODERN:
         bundleFunction = (callback) => this._modernBundle(callback, bundleType);
         break;
+      case BundleType.SEARCH_BAR_LEGACY_IIFE:
```

Comment:
nit: What are your thoughts on changing from a switch statement to a multiple if statements? I feel like that would be simpler

Reply:
I may just not be as used to reading switch statements, but up to you. 

I was thinking something like this:

```
const modernBundles = [BundleType.MODERN, BundleType.SEARCH_BAR_MODERN];
const iifeBundles = [BundleType.LEGACY_IIFE, BundleType.SEARCH_BAR_LEGACY_IIFE];
const legacyBundles = [BundleType.LEGACY_UMD, BundleType.SEARCH_BAR_LEGACY_UMD];

if (modernBundles.includes(bundleType)) {
  bundleFunction = (callback) => this._modernBundle(callback, bundleType);
} else if (iifeBundles.includes(bundleType)) {
  bundleFunction = (callback) => this._legacyBundleIIFE(callback, bundleType);
} else if (legacyBundles.includes(bundleType)) {
  bundleFunction = (callback) => this._legacyBundleUMD(callback, bundleType);
} else {
  throw new Error('Unrecognized BundleType');
}
```

Reply:
Done!